node/runtime/src/test_utils.rs

```diff
@@ -392,9 +392,22 @@ impl User {
 pub fn setup_test_contract(wasm_binary: &[u8]) -> (User, CryptoHash) {
     let (runtime, state_db, genesis_root) = get_runtime_and_state_db();
     let (mut user, root) = User::new(runtime, "alice.near", state_db.clone(), genesis_root);
-    let (new_root, _) = user.deploy_contract(
+    let (root_with_account, _) = user.create_account(root, "test_contract", 0);
```

Comment:
We should give this contract some money, even if it's just a test :)

Reply:
Lol, why?

Reply:
We can expose it later, once needed